nchelpers/__init__.py

```diff
@@ -8,13 +9,51 @@
 import numpy as np
 from nchelpers.date_utils import resolution_standard_name, time_to_seconds, d2ss
 
```

Comment:
@jameshiebert do you think these helper-ish functions should become @classmethods?